protocol/triple/internal/server/cmd_instance_with_registry/main.go

```diff
@@ -31,11 +31,11 @@ func main() {
 	// configure global configurations and common modules
 	ins, err := dubbo.NewInstance(
 		dubbo.WithName("dubbo_test"),
-		dubbo.WithRegistry("zk",
+		dubbo.WithRegistry(
 			registry.WithZookeeper(),
 			registry.WithAddress("127.0.0.1:2181"),
```

Comment:
Is 'registry.WithAddress("zookeeper://127.0.0.1:2181")' a valid method to configure Registry, a combination of WithZookeeper() and WithAddress("127.0.0.1:2181")? We should support this format if we want to give a consistent experience to users.